server/cmd/mmctl/commands/import.go

```diff
@@ -112,6 +112,7 @@ func init() {
 	ImportValidateCmd.Flags().Bool("check-missing-teams", false, "Check for teams that are not defined but referenced in the archive")
 	ImportValidateCmd.Flags().Bool("ignore-attachments", false, "Don't check if the attached files are present in the archive")
 	ImportValidateCmd.Flags().Bool("check-server-duplicates", true, "Set to false to ignore teams, channels, and users already present on the server")
+	ImportValidateCmd.Flags().String("data-path", "", "The base path for files attached to the export")
```

Comment:
I notice the app changes seem to be specific to user related data, but the imports can deal with more files (e.g. attachments, emojis). Are these treated differently? I guess I am a little confused and worried we may be adding more inconsistencies here so I'd welcome a full review of anything expecting a path.